### DISCUSSION TITLE
[Enhancement] [tree-select] 迫切希望给select、treeselect 等下拉选择组件添加一个值渲染的插槽，便于自定义个性化的显示

### DISCUSSION THREAD

**Discussion by jinkunaier** [element-plus/element-plus#10034] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `tree-select`

## Description

迫切希望给 el-select ,treeselect 组件添加值渲染的插槽，那样表单回显时，对于异步加载的树可以自已去处理回显内容，还可以自定义一些个性化的显示。类似的功能在antd-vue中treeselect有tagRender插槽 ,tdesign的treeselect有valueDisplay插槽。

### Additional comments

_(empty)_

**Comment by jinkunaier** [comment]
大家都顶起来啊，太需要这个功能了！！！！

**Comment by YujiaCheng1996** [comment]
在用tree-v2，但是高度写死了，不能自定义。。。

**Comment by lhz239996423** [comment]
我也需要这个功能，目前select非常需要支持一个输入插槽
